packages/server-renderer/src/renderToString.ts

```diff
@@ -142,12 +144,21 @@ function renderComponentSubTree(
       setCurrentRenderingInstance(null)
     } else if (comp.render) {
       renderVNode(push, renderComponentRoot(instance), instance)
+    } else if (comp.template && isString(comp.template)) {
+      const compileResult = compile(comp.template, {
+        isCustomElement: instance.appContext.config.isCustomElement || NO
```

Comment:
We should also handle caching and `onError` reporting similar to [here](https://github.com/vuejs/vue-next/blob/master/packages/vue/src/index.ts#L10).